package.json

```diff
@@ -26,9 +26,9 @@
   },
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.9.0",
-    "@openshift/dynamic-plugin-sdk": "^1.0.0",
-    "@openshift/dynamic-plugin-sdk-extensions": "^1.0.0",
-    "@openshift/dynamic-plugin-sdk-utils": "^1.0.0",
+    "@openshift/dynamic-plugin-sdk": "^2.0.1",
+    "@openshift/dynamic-plugin-sdk-extensions": "^1.1.0",
+    "@openshift/dynamic-plugin-sdk-utils": "^1.1.1",
```

Comment:
It errors out if I import and use the `useWorkspace` hook. After some investigation, it looks like
Webpack always servers 1.1.0 version of the utils (used in hac-core's [package.json ](https://github.com/openshift/hac-core/blob/main/frontend/package.json#L31)) and `dynamic-plugin-sdk-utils` is declared as a shared singleton module [hac-core/plugin.ts](https://github.com/openshift/hac-core/blob/main/frontend/config/plugins.js#L20), so we might need to update in hac-core as well?

loads old/1.1.0 version:

<img width="921" alt="image" src="https://user-images.githubusercontent.com/9964343/217756126-323c5deb-34ac-460e-ae2c-16bd8a4371fe.png">

Reply:
For me it crashes even without importing the hook
```
ErrorWithCause: Failed to load module 'Applications' in extension hac-dev[19]_e12
...
Caused by: Error: Shared module redux doesn't exist in shared scope default
```
![screenshot-1675941997](https://user-images.githubusercontent.com/20013884/217800182-ef619214-6c23-4a88-ae72-e859a8e49899.png)

Reply:
@rottencandy redux issue is fixed already - [slack thread](https://redhat-internal.slack.com/archives/C038DJAP7HR/p1675938562913009?thread_ts=1675935775.289279&cid=C038DJAP7HR).

Reply:
PR here: https://github.com/openshift/hac-core/pull/132